### PR TITLE
Patch request completed with error handling added

### PR DESCRIPTION
**Pull request by CRJenner**
Patch request committed with updated reviews

error added for non numbers. incorrect path already done in past error.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -76,6 +76,41 @@ describe("app", () => {
         });
     });
   });
+  describe("PATCH /api/reviews/:review_id", () => {
+    test("200: Patch returns the updated article", () => {
```
Comment (aloush-dev) [inline __tests__/app.test.js]: Isn't the test just the same as the one below?

**Review by aloush-dev — APPROVED** [review]
Nice work.
I'd also test for and code, what if you're passed in an empty object or something other than inc_votes?

Then please merge away!
